lib/sentry/client.ex

```diff
@@ -31,6 +31,13 @@ defmodule Sentry.Client do
     end)
   end
 
+  def dsn_env do
```

Comment:
Unfortunately it is public for testing, this should be moved to another module `DsnParser` and be injected on `send_event` public function